src/components/scene/inspector.js

```diff
@@ -12,12 +12,35 @@ module.exports.Component = registerComponent('inspector', {
   },
 
   init: function () {
+    this.loadingMessage = 'Loading Inspector';
```

Comment:
this could just be a constant rather than a property of the component

`var LOADING_MESSAGE`